Bug: [sflow] test_sflow.py setup fails if VLAN1000 is not populated

### Is it platform specific

generic

### Importance or Severity

Medium

### Description of the bug

sflow/test_sflow.py has a line of setup code that tries to remove entries from a VLAN which will fail if those entries don't already exist. It should instead just ignore those errors, since all it's doing is removing existing entries so that it can add it's own: https://github.com/sonic-net/sonic-mgmt/blob/4082f1c4ed254713508e2908dc3a84c74a81b6af/tests/sflow/test_sflow.py#L120

### Steps to Reproduce

Run `sudo config vlan member del 1000 Ethernet224` then run the test and you will see:
```
E           tests.common.errors.RunAnsibleModuleFail: run module command failed, Ansible Results =>
E           failed = True
E           changed = True
E           rc = 2
E           cmd = ['config', 'vlan', 'member', 'del', '1000', 'Ethernet224']
E           start = 2025-11-03 21:33:30.397668
E           end = 2025-11-03 21:33:30.580893
E           delta = 0:00:00.183225
E           msg = non-zero return code
E           invocation = {'module_args': {'_raw_params': 'config vlan member del 1000 Ethernet224', '_uses_shell': False, 'warn': False, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}
E           _ansible_no_log = None
E           stdout =
E           stderr =
E           Usage: config vlan member del [OPTIONS] <vid> <port>
E           Try "config vlan member del -h" for help.
E
E           Error: Ethernet224 is not a member of Vlan1000
```

### Actual Behavior and Expected Behavior

Run the test when VLAN1000 is not pre-configured with entries.

Actual: test fails during setup

Expected: test passes

### Relevant log output

```shell

```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/sflow/test_sflow.py
+++ b/tests/sflow/test_sflow.py
@@ -117,7 +117,7 @@ def config_dut_ports(duthost, ports, vlan):
     # Even though port is deleted from vlan , the port shows its master as Bridge upon assigning ip address.
     # Hence config reload is done as workaround. ##FIXME
     for i in range(len(ports)):
-        duthost.command('config vlan member del %s %s' % (vlan, ports[i]))
+        duthost.command('config vlan member del %s %s' % (vlan, ports[i]), module_ignore_errors=True)
         duthost.command('config interface ip add %s %s/24' %
                         (ports[i], var['dut_intf_ips'][i]))
     duthost.command('config save -y')

